docker/readmodel-db/purpose.sql

```diff
@@ -0,0 +1,63 @@
+CREATE SCHEMA IF NOT EXISTS readmodel_purpose;
+
+CREATE TABLE IF NOT EXISTS readmodel_purpose.purpose (
+  id UUID,
+  metadata_version INTEGER NOT NULL,
+  eservice_id UUID NOT NULL,
+  consumer_id UUID NOT NULL,
+  delegation_id UUID,
+  suspended_by_consumer BOOLEAN,
+  suspended_by_producer BOOLEAN,
+  title VARCHAR NOT NULL,
+  description VARCHAR NOT NULL,
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL,
+  updated_at TIMESTAMP WITH TIME ZONE,
+  is_free_of_charge BOOLEAN NOT NULL,
+  free_of_charge_reason VARCHAR,
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE IF NOT EXISTS readmodel_purpose.purpose_risk_analysis_form (
+  id UUID,
+  purpose_id UUID NOT NULL REFERENCES readmodel_purpose.purpose (id) ON DELETE CASCADE,
+  metadata_version INTEGER NOT NULL,
+  version VARCHAR NOT NULL,
+  risk_analysis_id UUID,
```

Comment:
Align with the risk analysis schema used in e-service, they should be the same (there you have risk_analysis_form_id/risk_analysis_form_version, here you use a different naming, and the references/tables differ a little bit)